<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/aaa.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_entites">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>741</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Blackadder ITC</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 225);
color: rgb(85, 85, 127);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_Promotions">
    <attribute name="title">
     <string>Promotions</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_promotions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(71, 142, 255);
font: italic 12pt &quot;MS Sans Serif&quot;;
</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_promotions_ajouter">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pushButton__valider_promotions_ajouter">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 255, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mail_promotion">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>330</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/mail2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>381</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_Promotions_Ajouter">
        <item row="0" column="0">
         <widget class="QLabel" name="label_ID_promotions_ajouter">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ID_promotions_ajouter"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Nom_promotions_ajouter">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_Nom_promotions_ajouter"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_pourcentge_promotions_ajouter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>Pourcentage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_pourcentge_promotions_ajouter"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_promotions_consulter">
      <attribute name="title">
       <string>Consulter</string>
      </attribute>
      <widget class="QLabel" name="label_liste_promotions_consulter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 255, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Liste des promotions</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_Promotions_consulter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>501</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ID_promotion_supprimer">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>310</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_supp_promotions">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>310</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp_promotions">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>360</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/supprimer.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modif_promotions">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>130</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/modifier.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Packs">
    <attribute name="title">
     <string>Packs</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_packs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(71, 142, 255);
font: italic 12pt &quot;MS Sans Serif&quot;;


</string>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_packs_ajouter">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_valider_Packs_ajouter">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 255, 190, 255), stop:1 rgba(255, 255, 255, 255));
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>301</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_pack">
        <item row="0" column="0">
         <widget class="QLabel" name="label_ID_Packs_Ajouter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>ID_Pack</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Categorie_Packs_Ajouter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>Categorie</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Liste_Packs_Ajouter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>Liste</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_Liste_Packs_Ajouter"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_ID_Packs_Ajouter"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_prix_initial_packs_ajouter">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="text">
           <string>Prix_initial</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_prix_initial_packs_ajouter"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_Categorie_Packs_Ajouter">
          <item>
           <property name="text">
            <string>Silver</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gold</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Premium</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_packs_consulter">
      <attribute name="title">
       <string>Consulter</string>
      </attribute>
      <widget class="QLabel" name="label_liste_packs_consulter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 255, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>Liste des packs</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rechercher_packs_consulter">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>80</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_packs_consulter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>471</width>
         <height>361</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="ID_pack_supprimer">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>320</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(79, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="text">
        <string>ID_Pack</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sup_pack">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>320</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp_pack">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>360</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/supprimer.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modif_pack">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>200</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/modifier.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_rechercher_packs">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>80</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/rechercher.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_pack_ID">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>50</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/images/tri_AZ.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_categorie_pack">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>120</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string> tri categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_pack_prix">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>tri prix</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_id_pack_inverse">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>trier id inv</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_categorie_pack_inverse">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>220</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>tri categorie inv</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_tri_prix_pack_inverse">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>260</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>tri prix inv</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_packs_statistiques">
      <attribute name="title">
       <string>Statistiques</string>
      </attribute>
      <widget class="QWidget" name="widget_green" native="true">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>30</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 0);</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_red" native="true">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>120</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_blue" native="true">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>210</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Ventes">
    <attribute name="title">
     <string>Ventes</string>
    </attribute>
    <widget class="QLabel" name="label_liste_Ventes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 255, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="text">
      <string>Liste des ventes</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_Ventes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>701</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
